rero_ils/modules/patrons/api.py

```diff
@@ -217,8 +218,7 @@ def update(self, data, dbcommit=False, reindex=False):
         return self
 
     @classmethod
-    def _anonymize_loans(
-            cls, patron_data=None, old_keep_history=None):
+    def _anonymize_loans(cls, patron_data=None, old_keep_history=None):
         """Anonymize patron loans.
 
         :param patron_data - dictionary representing a patron record.
```

Comment:
Is it
```suggestion
        :param patron_data: dictionary representing a patron record.
```
?

Oh, it's not your change. Sorry. :slightly_smiling_face: